Automattic/jetpack
# Add default values for jetpack forms' CSS veriables

## Proposed changes:

This patch fixes the issue of layout shifts that occur on form elements after the page has loaded due to the delay before JS can run.

This is done by providing sensible defaults for our CSS variables as suggested by @jcheringer in #29140. Because the root cause was the delay before JS could run, not slow loading of the CSS itself, that PR is now obsolete.

This PR also removes the obnoxious blur effect on load which is no longer necessary for the same reason as described above.

### Other information:


## Jetpack product discussion



-

## Does this pull request change what data or activity we track or use?



No.

## Testing instructions:







- Create a post with a contact form or forms.
- Test for either style variation: default, outlined and animated.
- Publish the post.
- When you view the public post, the contact form blocks should no longer be blurred while they're loading.
- While the page is loading, the form inputs and labels should also 'not shift violently'.
